pkg/chart/dependency_test.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
why changed? please undo those changes.

Reply:
This modification is execute `gofmt -s` automatic output format adjustment

Reply:
All changes of this type have been reverted